packages/transport/src/modules/publicRelationshipTemplatesReference/PublicRelationshipTemplateController.ts

```diff
@@ -0,0 +1,37 @@
+import { RequestError } from "../../core/backbone/RequestError";
+import { ControllerName, TransportController } from "../../core/TransportController";
+import { AccountController } from "../accounts/AccountController";
+import { PublicRelationshipTemplateReferenceClient } from "./backbone/PublicRelationshipTemplateReferenceClient";
+import { PublicRelationshipTemplateReference } from "./data/PublicRelationshipTemplateReference";
+
+export class PublicRelationshipTemplateReferencesController extends TransportController {
+    public constructor(parent: AccountController) {
+        super(ControllerName.PublicRelationshipTemplateController, parent);
```

Comment:
Do you want to name the Controller `PublicRelationshipTemplateReferencesController` or `PublicRelationshipTemplateController`? There is an inconsistency currently. Since `PublicRelationshipTemplateReferencesController` occurs more frequently and the facade is also called like this, I would assume that it should be called `PublicRelationshipTemplateReferencesController`. 

Please use search functions to check that the other variant (`PublicRelationshipTemplateController`) is changed to the correct variant.